docs/MapColonies/DEM/Services/elevation/README.md

```diff
@@ -0,0 +1,89 @@
+---
+id: dem-elevation-service
+slug: info
+title: Elevation ⛰️
+description: Service to return geographic coordinate [lat, lon] elevation 
+tags:
+  - dem
+  - API
+  - elevation
+  - height
+  - coordinate
+---
+
+# Elevation Service ⛰️
+
+:::info
+Click [here](/docs/MapColonies/dem/services/elevation/api) for the OpenAPI
+:::
+
+## Purpose
+The service lets users to recieve elevation(height) info at a given location(s)/position(s) [lon,lat] with/without extra data eg. resolution, DTM/DSM, etc.
+
+## Terminology
+
+Elevation info might be returned from following materials:
+1. DTM - terrain materials.
+2. DSM - surface materials.
+3. MIXED - according to most detailed(best resolution) materials.
+
+## Usage
+
+
+:::caution
+**- You can ask elevation values for top 250 points at once. If you need more points to process you should devide it to bulks**
+**- Coordinates order is not preserved**<br/>
+**- You will need an API token as part of the service authentication &nbsp;**<br/>
+**- When getting a time-out in a result of request, please try same API call again.**
+:::
+
+
+The API in general is split to two parts:
+1. JSON payload - you can query the service with payload in JSON format.
+2. Comressed payload(protobuf) - you can query the the service with COMRESSED(protobuf) payload for performance reason. <br/>
+<i style={{color:'orange'}}>If you interested, contact us for explanation how to use it.</i>
```

Comment:
Please remove, currently we agreed that this will not be supported.

Reply:
Who agree? Why?

Reply:
After discussion with @TalZmiri @shimoncohen and @gal decided to remove it.

DONE